index.js

```diff
@@ -1 +1,130 @@
+const popup = document.querySelector(".popup");
 
+const inputFormAddTask = document.querySelectorAll(".form__input-task");
```

Comment:
Для переменных имеющих внутри списки/коллекции/массивы лучше использовать в названии list, например:
```suggestion
const formInputsList = document.querySelectorAll(".form__input-task");
```

Если у тебя найден popup, лучше искать не в глобальной области видимости, а в области видимости твоего попапа:
```suggestion
const inputFormAddTask = popup.querySelectorAll(".form__input-task");
```

В данном кейсе нет смысла искать через коллекцию, т.к. у тебя может быть только один title, так же как и тело, в будущем если будет необходимость добавляться поля ввода, нужно будет добавлять новые query selectors.